packages/www/components/Login/index.tsx

```diff
@@ -36,23 +43,95 @@ const Login = ({
           }
           const [hashedPassword] = await hash(password, FRONTEND_SALT);
           // hash password, then
-          onSubmit({ email, password: hashedPassword });
+          onSubmit({
+            email,
+            password: hashedPassword,
+            ...(firstName && { firstName }),
```

Comment:
does this have the desired results if firstName is empty?

Reply:
Removed ugly conditional syntax in [04ff25](https://github.com/livepeer/livepeer-com/pull/25/commits/04ff2537a397629316e3ccef0ffba6a97665ad0a)